JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R

```diff
@@ -195,26 +195,26 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
   
   # error check: Posterior too peaked?
   if(abs(CIlower - CIupper) <= .Machine$double.eps){
-    plot$setError("Plotting not possible: Posterior too peaked!")
+    plot$setError(gettext("Plotting not possible: Posterior too peaked!"))
     return()
   }
   
   ppCri           <- c(CIlower, CIupper)
   dfLinesPP       <- .dfLinesPP(dataset = NULL, a = a, b = b, hyp = hypothesis, theta0 = theta0, counts = successes, n = n)
   dfPointsPP      <- .dfPointsPP(dataset = NULL, a = a, b = b, hyp = hypothesis, theta0 = theta0, counts = successes, n = n)
-  xName           <- expression(paste("Population proportion", ~theta))
+  xName           <- expression(paste(gettext("Population proportion"), ~theta))
```

Comment:
expression will make you cry

Reply:
aka gettext must be removed here and later @vandenman  or @TimKDJ can help

Reply:
okay to fix this do:

```r
xName <- bquote(paste(.(gettext("Population proportion")), ~theta))
```
`bquote` is similar to expression but everything inside `.()` is evaluated.